lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -875,6 +873,9 @@ def __init__(self, asyncpg):
 
     def connect(self, *arg, **kw):
         async_fallback = kw.pop("async_fallback", False)
+        creator_fn = kw.pop(
```

Comment:
right, this has to be specific to each asyncio DBAPI

so...we would need asyncmy, aiosqlite, aiomysql also...

Reply:
Added supported for the three additional drivers